### ISSUE TITLE
create Analytics for OrgBook number of searches per session

### ISSUE THREAD

**Issue by CharlesMacpherson** [bcgov/orgbook-bc-client#206]
Dwayne Gordon has requested that we have analytics that show the number (and average number) of searches a user has per session. The idea is that we can help determine if the searches are successful.

I believe we have GDX analytics in place (Snowplow and Looker) on OrgBook.

**Comment by WadeBarnes** [comment]
Yes, we have GDX analytics in place (Snowplow).  It's available through the intranet and there are a handful of us that have access.  Myself, you, not sure who else.

**Comment by esune** [comment]
> Yes, we have GDX analytics in place (Snowplow). It's available through the intranet and there are a handful of us that have access. Myself, you, not sure who else.

@CharlesMacpherson please let us know if this is enough to resolve this issue, if Dwayne needs access to snowplow we might be able to just provide him with it?

**Comment by CharlesMacpherson** [comment]
Existing dashboards are here:
https://intranet.qa.gov.bc.ca/analytics/orgbook-bc-dashboard?embedId=E8AF91215E6244E9A65664E97460FB64
Dwayne, Charles, Wade King, Wade Barnes, John have access. The analytics is based on API calls, so there is nothing specific to the number of searches per user session. We'd need to make a request to GDX to include that, I believe.

**Comment by esune** [comment]
@CharlesMacpherson @WadeBarnes do you have indications on how/what to do to submit the request?

**Comment by WadeBarnes** [comment]
How, yes.  What, no.  But we might be able to have a meeting with the team at GDX and describe what we want, and they may know how to implement it.

**Comment by WadeBarnes** [comment]
@danpollock was our main point of contact when the Dashboard was first setup.  Dan, are you still the main point of contact?

**Comment by danpollock** [comment]
@WadeBarnes : Yes I am!

**Comment by WadeBarnes** [comment]
> @WadeBarnes : Yes I am!

Excellent!
